hieroplatform/src/main/java/org/hiero/sketch/table/Converters.java

```diff
@@ -3,22 +3,30 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 import java.time.Duration;
-import java.util.Date;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
 
 /**
  * Conversion to and from doubles of various supported datatypes.
  */
 class Converters {
-    public static double toDouble(@NonNull final Date d) {
-        return d.getTime();
+    public static final LocalDateTime baseTime = LocalDateTime.of(
```

Comment:
Please add a comment to this field; in essence we are converting dates to doubles by converting them to intervals from a fixed date and converting the intervals to doubles. This is fine as long as we never change the fixed date.